boot/assets_test.go

```diff
@@ -529,18 +542,15 @@ func (s *assetsSuite) TestInstallObserverObserveExistingRecoveryNoAssets(c *C) {
 func (s *assetsSuite) TestInstallObserverObserveExistingRecoveryReuseNameErr(c *C) {
 	d := c.MkDir()
 
-	tab := bootloadertest.Mock("recovery-bootloader", "").WithTrustedAssets()
-	bootloader.Force(tab)
-	defer bootloader.Force(nil)
-	tab.TrustedAssetsList = []string{
+	tab := s.bootloaderWithTrustedAssets(c, []string{
```

Comment:
Shouldn't we defer bootloader.Force(nil) after this call?

Reply:
that's being handled by this call inside bootloaderWithTrustedAssets:

```go
	s.AddCleanup(func() { bootloader.Force(nil) })
```